hw/ip/spi_device/data/spi_device_testplan.hjson

```diff
@@ -317,27 +318,30 @@
             - Check response to read command.
             - Check if command is processed internally.'''
       milestone: V2
-      tests: []
+      tests: ["spi_device_mailbox"]
     }
     {
       name: mailbox_cross_outside_command
       desc: '''
+            - Configure spi_device on passthrough mode. Mailbox boundary crossing is not expected to
+              be used on flash mode, so that testing this on flash mode isn't needed.
```

Comment:
@eunchan I think we agree on this, right?

Add @a-will, in case that we want to have mailbox boundary crossing on flash mode.

Reply:
Correct.

Reply:
Agreed.